scripts/featured

```diff
@@ -319,7 +319,20 @@ class FeatureHandler(object):
         Returns:
             None.
         """
-        restart_field_str = "always" if "enabled" in feature_config.auto_restart else "no"
+        # As per the current code(due to various dependencies) SWSS service stop/start also stops/starts the dependent services(syncd, teamd, bgpd etc)
+        # There is an issue seen of syncd service getting stopped twice upon a critical process crash in syncd service due to above reason.
+        # Also early start of syncd service has traffic impact on VOQ chassis.
+        # to fix the above issue, we are disabling the auto restart of syncd service as it will be started by swss service.
+        # This change can be extended to other dependent services as well in future and also on pizza box platforms.
+
+        device_type = self._device_config.get('DEVICE_METADATA', {}).get('localhost', {}).get('type')
+        is_dependent_service = feature_config.name in ['syncd', 'gbsyncd']
+        if device_type == 'SpineRouter' and is_dependent_service:
```

Comment:
We need to check if we are doing a warm boot before setting this, as we start the peer dependent services if we are doing a warm boot https://github.com/sonic-net/sonic-buildimage/blob/master/files/scripts/swss.sh#L237

Reply:
@judyjoseph as this check is for Spine route warm-boot is don;t care

Reply:
@judyjoseph yes. once we extend this for pizza box, will add warm-boot check accordingly.